docs/release-info/release-notes.md

```diff
@@ -73,47 +134,226 @@ This section contains detailed release notes separated by areas.
+
+[#17735](https://github.com/apache/druid/pull/17735)
+
+#### Other metrics and monitoring changes
+
+- Added the `ingest/processed/bytes` metric that tracks the total number of bytes processed during ingestion tasks for JSON-based batch, SQL-based batch, and streaming ingestion tasks [#17581](https://github.com/apache/druid/pull/17581)
+
 ### Extensions
 
+#### Kubernetes 
+
+- You can now ingest payloads larger than 128KiB when using HDFS as deep storage for Middle Manager-less ingestion [#17742](https://github.com/apache/druid/pull/17742)
+- You can now run task pods in a namespace different from the rest of the cluster [#17738](https://github.com/apache/druid/pull/17738)
+- You can now name your K8s job names using `druid.indexer.runner.k8sTaskPodNamePrefix` [#17749](https://github.com/apache/druid/pull/17749)
+- The logging level is now set to info. Previously, it was set to debug [#17752](https://github.com/apache/druid/pull/17752)
+- Druid now supports lazy loading of pod templates so that any config changes you make are deployed more quickly [#17701](https://github.com/apache/druid/pull/17701)
+- Removed startup probe so that peon tasks can start up properly without being killed by Kubernetes [#17784](https://github.com/apache/druid/pull/17784)
+
 ### Documentation improvements
 
 ## Upgrade notes and incompatible changes
 
 ### Upgrade notes
 
-#### Front-coded dictionaries
+#### `useMaxMemoryEstimates`
+
+`useMaxMemoryEstimates` is now set to false for MSQ task engine tasks. Additionally, the property has been deprecated and will be removed in a future release. Setting this to false allows for better on-heap memory estimation.
 
-<!--Carry this forward until 32. Then move it to incompatible changes -->
+[#17792](https://github.com/apache/druid/pull/17792)
 
-In Druid 32.0.0, the front coded dictionaries feature will be turned on by default. Front-coded dictionaries reduce storage and improve performance by optimizing for strings where the front part looks similar.
+#### Automatic kill tasks interval
 
-Once this feature is on, you cannot easily downgrade to an earlier version that does not support the feature. 
+Automatic kill tasks are now limited to 30 days or fewer worth of segments per task.
 
-For more information, see [Migration guide: front-coded dictionaries](./migr-front-coded-dict.md).
+The previous behavior (no limit on interval per kill task) can be restored by setting `druid.coordinator.kill.maxInterval = P0D`.
 
-If you're already using this feature, you don't need to take any action. 
+[#17680](https://github.com/apache/druid/pull/17680)
 
+#### Kubernetes deployments
 
-### Incompatible changes
+By default, the Docker image now uses the canonical hostname if you're running Druid in Kubernetes. Otherwise, it uses the IP address otherwise [#17697](https://github.com/apache/druid/pull/17697) 
```

Comment:
```suggestion
By default, the Docker image now uses the canonical hostname to register services in ZooKeeper for internal communication if you're running Druid in Kubernetes. Otherwise, it uses the IP address. [#17697](https://github.com/apache/druid/pull/17697). 

You can set the environment variable `DRUID_SET_HOST_IP` to `1` to restore old behaviour.
```